rdf2vec/walkers/anonymous.py

```diff
@@ -4,10 +4,35 @@
 from hashlib import md5
 
 class AnonymousWalker(RandomWalker):
+    """Defines the anonymous walker of the walking strategy.
+
+    Attributes:
+        depth (int): The depth per entity.
+        walks_per_graph (float): The maximum number of walks per entity.
+
+    """
+
     def __init__(self, depth, walks_per_graph):
         super(AnonymousWalker, self).__init__(depth, walks_per_graph)
 
     def extract(self, graph, instances):
+        """Extracts a knowledge graph and transform it into a 2D vector, based
+        on provided instances.
+
+        Note:
```

Comment:
Perhaps we could leave this note out for now, as it is duplicated a lot. And make sure we include it on some "getting started" page in our documentation?